apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx

```diff
@@ -60,13 +60,9 @@ const ConfirmLogin = ({ email, onConfirmLogin }: confirmLoginProps) => {
               {confirmLogin.title}
             </Typography>
             {email && (
-              <Typography
-                variant='body2'
-                mb={6}
-                dangerouslySetInnerHTML={{
-                  __html: confirmLogin.body(email),
-                }}
-              />
+              <Typography variant='body2' mb={6}>
+                {confirmLogin.body(email).replace(/<[^>]*>/g, '')}
```

Comment:
I would remove the html tags used in the [translation](https://github.com/pagopa/developer-portal/blob/f9fdc00/apps/nextjs-website/src/_contents/translations.ts) file, instead of using the replace